### PR TITLE
HTML + TOC fixes

### PR DESCRIPTION
**Pull request by nparavicini7**
Fixed a handful of issues found from deploying the current master branch locally. HTML issues in some big data notebooks, and other issues with Markdown headers not causing sections to show up properly in right side table of contents. Also got rid of one remaining inline TOC with broken hyperlinks and fixed a few typos.

### CODE REVIEW

**Review by priyankatuteja — APPROVED** [review]

**Comment by priyankatuteja** [comment]
- [ ] fix left guide (https://developers.arcgis.com/python/guide/unsupervised-ml-model-guide/)

File: guide/07-working-with-bigdata/06-summarizing-data.ipynb [hunk 1]
```diff
@@ -82,7 +82,7 @@
     "      <td><a href=\"https://enterprise.arcgis.com/en/portal/latest/use/geoanalytics-summarize-within.htm\"><p align=\"left\">summarize_within</p></a></td>\n",
```
Comment (jyaistMap) [inline guide/07-working-with-bigdata/06-summarizing-data.ipynb:82]: <p>All the text in this guide below the Note displays as <strong>bold. </strong>Do you see that on your side when previewing locally?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: guide/07-working-with-bigdata/07-use_proximity.ipynb [hunk 2]
```diff
@@ -47,7 +47,7 @@
     "      <td><a href=\"https://enterprise.arcgis.com/en/portal/latest/use/geoanalytics-trace-proximity-events.htm\"><p align=\"left\">trace_proximity_events</p></a></td>\n",
```
Comment (jyaistMap) [inline guide/07-working-with-bigdata/07-use_proximity.ipynb:47]: <p>All the text in the <strong>Create Buffers </strong>section displays as bold, including the paragraph text and the code text within the cells. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (jyaistMap) [inline guide/07-working-with-bigdata/07-use_proximity.ipynb:47]: <p>All the text in the <strong>Trace Proximity Events </strong>section displays as bold, including the paragraph text and the code text within the cells. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (jyaistMap) [reply guide/07-working-with-bigdata/07-use_proximity.ipynb:47]: image

Comment (jyaistMap) [reply guide/07-working-with-bigdata/07-use_proximity.ipynb:47]: Looks great now!

File: guide/07-working-with-bigdata/06-summarizing-data.ipynb (hunk 1 above)
Comment (jyaistMap) [reply guide/07-working-with-bigdata/06-summarizing-data.ipynb:82]: Perfect!

**Review by jyaistMap — APPROVED** [review]
Everything looks great in local build: No duplicated TOCs and all the right-hand navs work great!

**Pull request merged**